pkg/sources/gitlab/gitlab.go

```diff
@@ -616,7 +618,7 @@ func (s *Source) Enumerate(ctx context.Context, reporter sources.UnitReporter) e
 	// Report all repos if specified.
 	if len(repos) > 0 {
 		for _, repo := range repos {
-			unit := sources.CommonSourceUnit{ID: repo}
```

Comment:
Oh, I never even noticed that it used to work this way. This is nice to clean up!